api_spec.yaml

```diff
@@ -318,33 +322,40 @@ paths:
                       example: 360000000
                     customAccessTokenPayload:
                       $ref: '#/components/schemas/createJWTPayload'
```

Comment:
response does not need to have timeCreatedMs

Reply:
Changes made.